layouts/api/list.html

```diff
@@ -152,7 +154,12 @@ <h2 class="mb-2" id="{{ $anchorStr | anchorize }}">
 
                     <h3 class="mt-2">{{ i18n "overview" }}</h3>
                     <p>{{ $translate_action.description | default $endpoint.action.description | markdownify }}</p>
-
+                    <p>
+                      {{ $translate_action.description | default $endpoint.action.description | markdownify }}
```

Comment:
Just seeing this and thinking it shouldn't be duplicated, i think the one on line 156 needs to go.

Reply:
removed!